server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -217,8 +217,12 @@ public enum MetadataVersion {
     // Add ELR related supports (KIP-966).
     IBP_3_9_IV1(22, "3.9", "IV1", true),
 
+    // Bootstrapping transaction version 2
```

Comment:
Should we add a new MV just for bootstrapping a new feature or just piggyback on an existing one (e.g., IBP_3_9_IV1)?

Reply:
I think I am going to move this to 4.0 anyway, so we can move it to an existing one there.